### PR TITLE
Feat/user-notification-settings

### PR DESCRIPTION
**Pull request by panicmilos**

### CODE REVIEW

File: features/users/settings/notification-types/configureNotificationTypes.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+type ConfiguredNotificationTypes = {
+  codes: string[];
+};
+
+function configureNotificationTypesForUser(userId: string) {
+  return (notifications: ConfiguredNotificationTypes) => {
```
Comment (draganaF) [inline features/users/settings/notification-types/configureNotificationTypes.ts:8]: You can name this configurationTypes to be consistent with method name.

**Review by draganaF — CHANGES_REQUESTED** [review]
One small change required.

File: features/users/settings/notification-types/configureNotificationTypes.ts (hunk 1 above)
Comment (panicmilos) [reply features/users/settings/notification-types/configureNotificationTypes.ts]: Done.

**Review by draganaF — APPROVED** [review]
Great job!

**Pull request merged**